Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs

```diff
@@ -189,9 +190,17 @@ public DateTime Taken
 	}
 	public class ConsensusPerformanceCounter
 	{
-		public ConsensusPerformanceCounter()
+        /// <summary>Provider of date time functionality.</summary>
+        private readonly IDateTimeProvider dateTimeProvider;
+
+        /// <summary>
+        /// Initializes an instance of the object.
+        /// </summary>
+        /// <param name="dateTimeProvider">Provider of date time functionality.</param>
+        public ConsensusPerformanceCounter(IDateTimeProvider dateTimeProvider)
 		{
-            this._Start = DateTime.UtcNow;
+            this.dateTimeProvider = dateTimeProvider;
+            this._Start = this.dateTimeProvider.GetUtcNow();
```

Comment:
`this._Start` -> `this.Start`

Reply:
these are off topics, we could make many new PRs fixing code style, tabs and comments, but if the old code was bad, it is OK not to fix; it is not OK to introduce new bad

Reply:
ok yeah ofcourse